bin/moodle-plugin-ci

```diff
@@ -45,6 +45,9 @@ if (file_exists(__DIR__.'/../../../autoload.php')) {
     exit(1);
 }
 
+// Current version. Keep it updated on releases.
+$version = (new \SebastianBergmann\Version('3.2.6', dirname(__DIR__)))->getVersion();
```

Comment:
Thinking more, may be version needed to be defined in `composer.json` instead of php script, which can be parsed for version displaying purposes here? That might have implications though (e.g. on composer versioning).

Reply:
About to make it a constant on top of file, I'm ok with that, good idea.

About to add it to composer... I just followed what other packages out there are doing (phpunit...) and read some comments about keeping them apart from composer explicitly.